melodic/distribution.yaml

```diff
@@ -6251,6 +6251,16 @@ repositories:
       url: https://github.com/ose-support-ros/omronsentech_camera.git
       version: master
     status: developed
+  omt_coscan:
+    doc:
+      type: git
+      url: https://github.com/siyandong/CoScan.git
+      version: master
+    source:
+      type: git
+      url: https://github.com/siyandong/CoScan.git
```

Comment:
The source repository here needs a LICENSE file, and to have the license declared in the package.xml files.

Also, the directory layout is pretty non-standard for a ROS package.  That doesn't mean it won't work, but it will look odd to people accustomed to ROS.  To make it more like ROS, I'd suggest moving all of the package directories (src/co_scan, src/srbot_description, src/virtual_scan) into the top-level of the source repository, and then removing the dummy src/CMakeLists.txt .  That will make it look a lot more standard.

Reply:
I'm sorry for the non-standard layout. I have reorganized the directories and added a LICENSE file.

Reply:
The repository layout looks a lot more standard, thanks for iterating.  Also thanks for adding a LICENSE file.

The last thing missing is declaring the `<license>` tag in each of the package.xml files.  Since your top-level LICENSE file is MIT, that would look like:

```
<license>MIT</license>
```

in each of the three package.xml files.  Once that is done, I'm happy to approve and merge.

Reply:
I add the license declaration in three .xml files. Thanks for your approval!